runtime/src/bank.rs

```diff
@@ -3757,9 +3756,7 @@ impl Bank {
     pub fn commit_transactions(
         &self,
         sanitized_txs: &[SanitizedTransaction],
-        mut processing_results: Vec<TransactionProcessingResult>,
-        last_blockhash: Hash,
-        lamports_per_signature: u64,
+        processing_results: Vec<TransactionProcessingResult>,
```

Comment:
not sure if im flying too close to the sun with this one and we should preserve the existing interface of `commit_transactions` even if we no longer need these params

Reply:
Nope, this is great. Was looking forward to these params getting removed